storage/pruning/fullHistoryTriePruningStorer.go

```diff
@@ -20,38 +19,38 @@ func NewFullHistoryTriePruningStorer(args *FullHistoryStorerArgs) (*fullHistoryT
 }
 
 func initFullHistoryTriePruningStorer(args *FullHistoryStorerArgs, shardId string) (*fullHistoryTriePruningStorer, error) {
-	err := checkArgs(args.StorerArgs)
-	if err != nil {
-		return nil, err
-	}
-
-	activePersisters, persistersMapByEpoch, err := initPersistersInEpoch(args.StorerArgs, shardId)
+	fhps, err := initFullHistoryPruningStorer(args, shardId)
 	if err != nil {
 		return nil, err
 	}
 
-	ps, err := initPruningStorer(args.StorerArgs, shardId, activePersisters, persistersMapByEpoch)
-	if err != nil {
-		return nil, err
-	}
-
-	tps := &triePruningStorer{ps}
-	ps.extendPersisterLifeHandler = tps.extendPersisterLife
-	tps.registerHandler(args.Notifier)
-
-	if args.NumOfOldActivePersisters < 1 || args.NumOfOldActivePersisters > math.MaxInt32 {
-		return nil, storage.ErrInvalidNumberOfOldPersisters
-	}
+	tps := &triePruningStorer{fhps.PruningStorer}
```

Comment:
multiline + add the member name
Hint: instead of having `fhps *FullHistoryPruningStorer` you can create an unexported interface and use a mock in the tests :D

Reply:
Done.